HacktoberfestProject.Web/Startup.cs

```diff
@@ -63,6 +63,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseAuthentication();
             app.UseAuthorization();
 
+            app.Use(async (ctx, next) =>
+            {
+                ctx.Response.Headers.Add("Content-Security-Policy", "connect-src 'self' https://api.github.com;");
```

Comment:
I think we should add a `default-src` as well:

>Your policy should include a `default-src` policy directive, which is a fallback for other resource types when they don't have policies of their own.

Source: [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)

Reply:
sounds sensible. @SekharV are you okay to make the change?

Reply:
Done @Layla-P